src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -41,8 +41,11 @@ public class AddCommand extends Command {
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "owesMoney";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This client already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New client added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This client already exists in the Realodex";
+    public static final String MESSAGE_ADD_HELP = "Add Command: Adds a client to the Realodex. You may input client"
+            + " details in any order. Tags are optional to input.\n"
+            + "Format: add n/NAME p/PHONE i/INCOME e/EMAIL a/ADDRESS f/FAMILY [t/TAG]...\n";
```

Comment:
Perhaps you can replace` f/FAMILY `with `f/FAMILYSIZE`? it is a little misleading as to what "family" is

Reply:
Agreed

Reply:
ps i will do these changes in another PR i think theres a bit to change but lets merge this first